src/Avalonia.Base/ExecuteActionDisposer.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Avalonia
+{
+    public class ExecuteActionDisposer : IDisposable
```

Comment:
Should be internal.
Or, for the Action you can use `Disposable.Create()` from the Avalonia.Reactive namespace.